config/environments/development.rb

```diff
@@ -33,6 +34,8 @@
     config.cache_store = :null_store
   end
 
+  config.logger = ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new($stdout))
```

Comment:
Your PR does not explain why you are making this change, which affects all users not just docker users.